rep-2000.rst

```diff
@@ -172,6 +172,122 @@ Middleware Implementation Support:
 - RTI Connext
 - ADLink OpenSplice
 
+Crystal Clemmys (December 2018 - TBD, at least December 2019)
+-------------------------------------
+
+Support Tiers:
+
+Tier 1
+
+Tier 1 platforms are subjected to our unit test suite and other testing tools on a frequent basis including continuous integration jobs, nightly jobs, packaging jobs, and performance testing.
+Errors or bugs discovered in these platforms are prioritized for correction by the development team.
+Significant errors discovered in Tier 1 platforms can impact release dates and we strive to resolve all known high priority errors in Tier 1 platforms prior to new version releases.
+
+Tier 2
+
+Tier 2 platforms are subject to ad hoc testing and binary packages may not be provided or supported.
+Tier 2 platforms may have downloadable binary archives
+However users should note that errors may be present in released product versions for Tier 2 platforms and, subject to resource availability, known errors in Tier 2 platforms may or may not be corrected prior to new version releases.
+
+Tier 3
+
+All platforms not specifically listed above.
+We neither run the unit test suite nor perform any other tests on platforms not listed above.
+We expect installation instructions to be present and up-to-date in order for a platform to be listed in this category.
+Other community members might provide assistance with these platforms.
+
+Targeted platforms:
+
++--------------+-----------------------+------------------------------+------------------------+---------------------------+------------------------+
+| Architecture | Ubuntu Bionic (18.04) |     MacOS Sierra (10.12)     |   Windows 10 (VS2017)  | Ubuntu Xenial (16.04) [s] | Debian Stretch (9) [s] |
++==============+=======================+==============================+========================+===========================+========================+
+|    amd64     |        Tier 1         |           Tier 1*            |          Tier 1*       |           Tier 2          |        Tier 3          |
```

Comment:
I'm not sure it makes sense to list Stretch as a Tier3. It seems like if we're going to list it it should be a Tier 2. Similar to xenial we expect it to work from source, but we're not building binaries for it.

Reply:
I believe the main distinction between Stretch and Xenial is that we have periodic CI jobs for Xenial

https://ci.ros2.org/view/nightly/job/nightly_xenial_linux_release/
https://ci.ros2.org/view/nightly/job/nightly_xenial_linux-aarch64_release/

 but not for Stretch. While we state that Tier 3 platforms are all those not mentioned I don't see a problem including Debian Stretch as the "reference" Tier 3 platform. I don't think we're in a position to elevate Stretch to a Tier 2 platform meeting the same level of coverage as Xenial has for the this release.